BUILDING.md

```diff
@@ -172,13 +172,21 @@ If you want to build DLLs, run `cmake` with `-DBUILD_SHARED_LIBS=ON`.
 
 ### Dependency management
 
-gRPC's CMake build system provides two modes for handling dependencies.
+gRPC's CMake build system has two options for handling dependencies.
+CMake can build the dependencies for you, or it can search for libraries
+that are already installed on your system and use them to build gRPC.
+
+This behavior is controlled by the `gRPC_<depname>_PROVIDER` CMake variables,
+ie `gRPC_CARES_PROVIDER`. The options that these variables take are as follows:
+
 * module - build dependencies alongside gRPC.
```

Comment:
nit: add "(the source code will be obtained from the git submodules)"?

Reply:
👍